Content.Shared/Movement/Systems/SharedMoverController.cs

```diff
@@ -365,18 +368,18 @@ public bool UseMobMovement(EntityUid uid)
     /// </summary>
     private bool IsAroundCollider(SharedPhysicsSystem broadPhaseSystem, TransformComponent transform, MobMoverComponent mover, EntityUid physicsUid, PhysicsComponent collider)
     {
-        var enlargedAABB = _lookup.GetWorldAABB(physicsUid, transform).Enlarged(mover.GrabRangeVV);
+        var enlargedAABB = new Box2Rotated(_lookup.GetWorldAABB(physicsUid, transform).Enlarged(mover.GrabRangeVV));
 
```

Comment:
Turning a Box2 into Box2Rotated doesn't make sense?

Reply:
It wanted Box2Rotated as an input :cry: